### PR TITLE
Set the default server url if CLI starts with no args

### PR DESCRIPTION
**Pull request by hjafarpour**
Currently we require ksql server url to be specified when starting KSQL cli. To keep the quick start and integration with confluent CLI simple this PR will use `http://localhost:8080` as the default server url when the cli is called without any args. If there is args in starting the cli the server url should be specified.

### CODE REVIEW

File: ksql-cli/src/main/java/io/confluent/ksql/Ksql.java [hunk 1]
```diff
@@ -35,6 +35,9 @@
 public class Ksql {
 
   public static void main(String[] args) throws IOException {
+    if (args.length == 0) {
```
Comment (dguy) [inline ksql-cli/src/main/java/io/confluent/ksql/Ksql.java]: `final Options options = args.length == 0 ? Options.parse("http://localhost:8080") : Options.parse(args);`

Comment (miguno) [reply ksql-cli/src/main/java/io/confluent/ksql/Ksql.java]: Question: Have we settled on `http://localhost:8080` as the default for the KSQL server endpoint?   Are 100% sure that it doesn't clash with any other CP components that might be running on the same machine when you do the CP quickstart?

Comment (hjafarpour) [reply ksql-cli/src/main/java/io/confluent/ksql/Ksql.java]: It does not have any problem working with other services in CP. But 8080 is kinda common port, it's default for tomcat, and may conflict with third party systems.

Comment (miguno) [reply ksql-cli/src/main/java/io/confluent/ksql/Ksql.java]: @hjafarpour : Would you then recommend to change the KSQL default listener port from `8080` to something else, like `8090`?  Or keep it as `8080`?

**Pull request merged**